ufs/session.go

```diff
@@ -0,0 +1,357 @@
+package ufs
```

Comment:
What does `ufs` stand for? Is this just from the other project?

Reply:
Unix filesystem